frontends/election-manager/src/components/election_manager_tally_report.tsx

```diff
@@ -138,15 +162,17 @@ export const ElectionManagerTallyReport = forwardRef<HTMLDivElement, Props>(
                       election={election}
                     />
                   </Prose>
+                  {manualTallyForReport && <ManualResultsNotice />}
```

Comment:
Personally I think it might be a little bit clearer to just have a flag here to figure out if there is a manualTally to determine whether or not to show the notice and then still pass all the externalTallies into ContestTally and have ContestTally do the logic of splitting out the manual tally inside of it, so that the caller doesn't always have to do that gymnastics, but I don't feel that strongly if you think its easier this way.

Reply:
I'm remembering now - I did it this way because the tally reports are already doing the gymnastics of filtering the results for the contest (which seems fine to me). So at the report level, we're filtering the `FullExternalExternalTallies` into `ExternalTallies`. `ExternalTallies` don't carry the source information with them, so I couldn't filter on those to identify the manual ones. I could edit the `ExternalTally` type to include the source. Or leave it as-is. Thoughts?